.github/workflows/ci.yml

```diff
@@ -17,15 +17,18 @@ jobs:
       - uses: dtolnay/rust-toolchain@stable
       - uses: Swatinem/rust-cache@v2
 
-      # Build the packages
-      - run: cargo build
-      - run: cargo build --features=custom-objective-percentile,custom-objective-latency
-
       # Run the tests with each of the different metrics libraries
-      - run: cargo test --features=prometheus-exporter
-      - run: cargo test --no-default-features --features=prometheus-exporter,metrics --tests
-      - run: cargo test --no-default-features --features=prometheus-exporter,prometheus --tests
-      - run: cargo test --no-default-features --features=prometheus-exporter,prometheus-client --tests
+      - run: cargo test --features=prometheus-exporter,metrics
+      - run: cargo test --features=prometheus-exporter,prometheus --tests
+      - run: cargo test --features=prometheus-exporter,prometheus-client --tests
+      # The tests using opentelemetry are currently failing because of this issue:
+      # https://github.com/open-telemetry/opentelemetry-rust/issues/1070
+      # We should re-enable them once 0.20 or 0.19.1 are released
+      # - run: cargo test --features=prometheus-exporter,opentelemetry
```

Comment:
maybe we can keep it enabled as a separate build test but allow it to fail (`continue-on-error: true`) so we don't forget about it

Reply:
Neat, didn't know about that